.env.example

```diff
@@ -1,3 +1,4 @@
 FLASK_ENV=development
 DATABASE_URI=postgresql://YOUR_USER:YOUR_PASSWORD@localhost:5432/YOUR_DATABASE
 SECRET=
+SMS_URI_KEY=
```

Comment:
Add a new line in the end of file.

Reply:
Probably this is due because of the lack of using `black .`

Reply:
done